easybuild/easyconfigs/s/StaMPS/StaMPS-3.3b1-intel-2017a-Perl-5.24.1.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'ConfigureMake'
+
+name = 'StaMPS'
+version = '3.3b1'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://homepages.see.leeds.ac.uk/~earahoo/stamps/'
+description = """A software package to extract ground displacements from time series of synthetic aperture radar
+ (SAR) acquisitions."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['http://homepages.see.leeds.ac.uk/~earahoo/stamps/']
+sources = ['%(name)s_v%(version)s.tar.gz']
+
+dependencies = [
+    ('Perl', '5.24.1'),
+    ('ROI_PAC', '3.0.1', versionsuffix),
+    ('Doris', '4.02'),
+    ('Triangle', '1.6'),
+    ('snaphu', '1.4.2'),
+]
+
+skipsteps = ['configure']
+
+start_dir = 'src'
+
+buildopts = 'CC="$CXX" CFLAGS="$CXXFLAGS"'
+
+preinstallopts = 'cp -a ../bin %(installdir)s/bin && '
+installopts = 'INSTALL_DIR=%(installdir)s/bin && cd .. && cp -a {DORIS_SCR,matlab,ROI_PAC_SCR} %(installdir)s/'
```

Comment:
the `make install` copies stuff from different locations into `$INSTALL_DIR`, I don't see why we should replicate that via `files_to_copy`

it doesn't copy the `_*SCR` directories though, so that's done manually